.gitignore

```diff
@@ -66,6 +66,9 @@ local_settings.py
 db.sqlite3
 db.sqlite3-journal
 
+#config
+/src/risknet/config/conf.yaml
```

Comment:
why would you gitignore this? If you check the handler, it also accepts a path to an environment variable.